modules/backend/widgets/mediamanager/assets/js/mediamanager.js

```diff
@@ -732,7 +732,8 @@
             paramName: 'file_data',
             timeout: 0,
             headers: {},
-            createImageThumbnails: false
+            createImageThumbnails: false,
+            maxFilesize: this.options.maxFilesize,
```

Comment:
We should probably just populate this from an ini_get(post_max_size) instead

Reply:
I am unsure how i'd proceed with this part.

Reply:
Make sure you're using the [latest version of winter/storm](https://github.com/wintercms/storm/commit/8675a200e88fa03ba66696e54afacc0ee7caa5fd) and add `data-max-filesize="<?= e(\File::getMaxUploadSize()); ?>"` to https://github.com/wintercms/winter/blob/develop/modules/backend/widgets/mediamanager/partials/_body.php#L13